### PR TITLE
Add era to `Unbonded` event

### LINKED ISSUE #11749: nomination-pools: Add era to `Unbonded` event
In nomination-pools tests that test cases that has Era as part of the case, it is a good idea to add in the `assert_eq!` of `pool_events_since_last_call` the era as part of the `eq`.
E.g. in the [partial_unbond_max_chunks](https://github.com/paritytech/substrate/blob/master/frame/nomination-pools/src/tests.rs#L2099) and [this assert_eq!](https://github.com/paritytech/substrate/blob/master/frame/nomination-pools/src/tests.rs#L2132-L2141) could have something like:
```
	Event::Unbonded { member: 20, pool_id: 1, balance: 3, points: 3, era: 1 },
	Event::Unbonded { member: 20, pool_id: 1, balance: 1, points: 1, era: 2 }
```

### PR DESCRIPTION
**Pull request by hirschenberger**
fixes #11749

### CODE REVIEW

File: frame/nomination-pools/src/lib.rs [hunk 1]
```diff
@@ -1288,6 +1288,7 @@ pub mod pallet {
 			pool_id: PoolId,
 			balance: BalanceOf<T>,
 			points: BalanceOf<T>,
+			era: EraIndex,
```
Comment (kianenigma) [inline frame/nomination-pools/src/lib.rs:1291]: Update the docs to say what is `era` please? is it the era in which we unbonded, or the one in which we will be unlocked?

**Review by kianenigma — APPROVED** [review]

**Comment by kianenigma** [comment]
needs to merge master.

**Review by shawntabrizi — APPROVED** [review]
sounds good after kian's suggestions are fixed

**Comment by kianenigma** [comment]
any updates @hirschenberger?

**Comment by hirschenberger** [comment]
> any updates @hirschenberger?

Sorry @kianenigma  was busy the last days. I will will fix your suggestions asap.

**Review by kianenigma — APPROVED** [review]

**Comment by kianenigma** [comment]
bot merge

**Comment by kianenigma** [comment]
bot rebase

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**